css_tasks/task11/scss/_service.scss

```diff
@@ -0,0 +1,88 @@
+@import "variables";
+@import "mixins";
+.service {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-column-gap: 2rem;
+  padding-top: $standardTopPadding;
+}
+.service__element {
+  img {
+    width: 100%;
+    display: block;
+  }
+}
+details {
```

Comment:
it would be better to work with classes, what if you will have another details elements with completely other styles

Reply:
Done